File: DataSummitSolution/DataSummitUI/src/app/efModels/currency.ts

export class Currency {

	CurrencyId: number;
	Entity: string;
	Name: string;
	AlphabeticCode: string;
	NumericCode: string;
	MinorUnit: string;
	IsFundYesNo: boolean;
	CreatedDate?: Date | string;
	UserId?: number;

	constructor(currencyId?: number, entity?: string, name?: string,
		alphabeticCode?: string, numericCode?: string, minorUnit?: string,
		isFundYesNo?: boolean, createdDate?: Date | string, userId?: number)
	{
		this.CurrencyId = currencyId || 0;
		this.Entity = entity || "";
		this.Name = name || "";
		this.AlphabeticCode = alphabeticCode || "";
		this.NumericCode = numericCode || "";
		this.MinorUnit = minorUnit || "";
		this.IsFundYesNo = isFundYesNo || false;
		this.CreatedDate = createdDate || new Date(Date.now()) || null;
		this.UserId = userId || 0 || null;
	}
}

File: DataSummitSolution/DataSummitModels/DB/Currency.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace DataSummitModels.DB
{
    public partial class Currency
    {
        public short CurrencyId { get; set; }
        public string Entity { get; set; }
        public string Name { get; set; }
        public string AlphabeticCode { get; set; }
        public string NumericCode { get; set; }
        public string MinorUnit { get; set; }
        public bool IsFundYesNo { get; set; }
        public DateTime? CreatedDate { get; set; }
        public long? UserId { get; set; }
    }
}
